web/src/components/pages/contribution/listen/listen.tsx

```diff
@@ -130,7 +130,7 @@ class ListenPage extends React.Component<Props, State> {
     const { clips } = this.state;
     const clipIndex = this.getClipIndex();
 
-    clearInterval(this.playedSomeInterval);
```

Comment:
Why is it necessary to remove that line?

Reply:
This line is already present inside stop() method.

Reply:
D'oh! Good point 🙈